libpebble3/schema/io.rebble.libpebblecommon.database.Database/24.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 24,
-    "identityHash": "4debf03d994cda016c2cb7b40242c919",
```

Comment:
If you are adding a field then the database version should be bumped to 25 (migration 24 has already gone out) - but I don't think you need that if you remove the change to `NotificationEntity`